lyft/l5kit
# Lberg/refactor test suite

Refactor of L5Kit tests:

- drop `unittest`;
- move all artefacts into a folder `artefacts` so that we can simulate data loading. Add metas;
- remove `agents_mask` (now computed at runtime)
- remove all dependency to files in l5kit (config mainly)
- remove all hard-coded matrices
- remove all mock functions to get rasterizers (replaced with functions from l5kit finally)

This should make tests much easier to maintain and develop.

PS
Sorry for the very long PR folks, but breaking it down into a lot of small ones was not trivial because of data dependencies. hopefully, it will be the only one this long in a while :)
